src/components/DailyStatistics/TransactionHistoryCard.tsx

```diff
@@ -58,30 +64,36 @@ export const TransactionHistoryCardComponent: FunctionComponent<TransactionHisto
   transactionHistory,
   loading,
 }) => {
+  const { c13nt, i18nt, c13ntForUnit } = useTranslate();
   return (
     <Card style={styles.stats}>
       {!loading ? (
         transactionHistory.length !== 0 ? (
           transactionHistory.map((item) => (
             <View key={item.category}>
               <View style={styles.transactionText}>
-                <AppText style={styles.categoryName}>{item.name}</AppText>
+                <AppText style={styles.categoryName}>
+                  {c13nt(item.name)}
+                </AppText>
                 <View style={styles.transactionText}>
-                  <AppText style={styles.quantityText}>
-                    {item.quantityText}
+                  <AppText style={styles.quantity}>{item.quantity}</AppText>
+                  <AppText style={styles.unit}>
+                    {c13ntForUnit(item.unit)?.label}
```

Comment:
This assumes that all the unit labels are postfixes. Is that okay?

Reply:
in CheckoutSuccessCard, is using "POSTFIX". Is that ok if we use that?

Reply:
instead of doing this, I think you should pass it to `formatQuantityText`. remember to replace the label with the translated one

Reply:
updated